src/runtime/refiner.ts

```diff
@@ -1332,7 +1332,7 @@ export class KTExtracter {
     }
     const expr = filterTerms.length > 0 ? `${filterTerms.join(' && ')}` : 'true';
 
-    return `${locals.map(x => `${x}\n`).join('')}return ${expr}`;
+    return `${locals.map(x => `${x}\n`).join('')}${expr}`;
```

Comment:
...maybe: 
`${locals.join('\n')}\n${expr}`

Reply:
That will insert an extra newline if there are no locals (which is admittedly rare).

I'm not sure its worth the change but am happy to be convinced.

Reply:
You're right. My suggestion wouldn't work. This is ultimately cosmetic, but now I'm invested. Here's one more attempt: 

```
[...locals, `${expr}`].join('\n');
```

Reply:
Ha that's pretty nice. I'll put it in the new PR (ahh, I see your comment there now, thanks)